### DISCUSSION TITLE
Automatic Slug Creation

### DISCUSSION THREAD

**Discussion by stolinski** [supabase/supabase#17079] [Feature Requests]
Slug fields are a super common requirement. While doing this in code isn't a big issue and you can do it via triggers in Supabase, it would be nice to have either documentation on best practices for automatic slug fields or some kind of helper to have this be automatic.

**Comment by MildTomato** [comment]
Is this sort of how you would handle slugs in your db? @stolinski 
https://www.kdobson.net/2019/ultimate-postgresql-slug-function/
